### ISSUE TITLE
Suggesting: premium system

### ISSUE THREAD

**Issue by ItsDiino** [AlexzanderFlores/WOKCommands#129]
can you add a premium system to this, it would like in the command file like ping.js and you defined it like:
`name: 'ping',
category: 'Help', 
premium: true` 
and i think it would be a great feature

**Comment by wrssmithjr04** [comment]
You can always make a pr with it

**Comment by cursorsdottsx** [comment]
No, what's the point when you can do it yourself? It's fairly easy, and besides, there are lots of systems to implement premium memberships, and I don't think Alex would go through the trouble to implement one that's super flexible and adaptable...
